dev-haskell/stan/stan-0.2.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Haskell STatic ANalyser"
+HOMEPAGE="https://github.com/kowainik/stan"
+
+LICENSE="MPL-2.0"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
```

Comment:
`KEYWORDS="~x86"` is the problem. I hit some weird unresolved bugs last time I tried using that arch, so I just removed it from the whole repo until it was properly tested.